### PR TITLE
jormun: upgrade to flask 1.0

### PR DESCRIPTION
**Pull request by kinnou02**
flask 0.14 is is vulnerable: [CVE-2019-1010083](https://nvd.nist.gov/vuln/detail/CVE-2019-1010083)

### CODE REVIEW

File: source/eitri/requirements.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ retrying==1.2.3
 psycopg2
 clingon==0.1.4
 GeoAlchemy2==0.2.4
-SQLAlchemy==1.3.3
```
Comment (AurelienLP) [inline source/eitri/requirements.txt]: Why a downgrade here ? :'(

File: source/sql/requirements.txt [hunk 2]
```diff
@@ -4,5 +4,5 @@ Mako==0.9.1
 MarkupSafe==0.18
 SQLAlchemy==1.3.3
 Shapely==1.3.0
-alembic==0.6.2
-psycopg2
```
Comment (AurelienLP) [inline source/sql/requirements.txt:8]: Shouldn't we fix the version of psycopg2 ?

File: source/eitri/requirements.txt [hunk 3]
```diff
@@ -4,7 +4,7 @@ retrying==1.2.3
...
+SQLAlchemy==1.0.11
```
Comment (mbounabi) [inline source/eitri/requirements.txt]: Downgrade?

**Review by mbounabi — APPROVED** [review]

File: source/eitri/requirements.txt (hunk 3 above)
Comment (kinnou02) [reply source/eitri/requirements.txt]: oops, wrong line...

File: source/sql/requirements.txt (hunk 2 above)
Comment (kinnou02) [reply source/sql/requirements.txt:8]: we want to use the psycopg installed by the system to not have rebuild everything

**Review by AurelienLP — APPROVED** [review]

**Pull request merged**